### PR TITLE
add beacon to V2 API

### PR DESCRIPTION
**Pull request by jamesgorrie**
- Adds a beacon to the V2 API to test load performance.
- Add a `defaultToggleValues` to S3 so we can change them within 30 seconds in case it all goes belly up. There's a bit of an edge case here where the values will be overridden on deploy, but it allows us to turn things off and then fix them. 

TODO:
- [x] add a beacon on the work page

### CODE REVIEW

File: common/koa-middleware/withToggles.js [hunk 1]
```diff
@@ -1,3 +1,16 @@
+const fetch = require('isomorphic-unfetch');
+
+let defaultToggleValues = {};
+async function getDefaultToggleValues() {
+  try {
+    const defaultValues = await fetch('https://dash.wellcomecollection.org/toggles/defaultToggleValues.json')
+      .then(response => response.json());
+
+    defaultToggleValues =  defaultValues;
```
Comment (davidpmccormick) [inline common/koa-middleware/withToggles.js]: There's an extra space after the `=`! I'll have a word with our linter config.

**Comment by jamesgorrie** [comment]
>  Is the v1 API currently only pinged on componentDidMount and componentDidUpdate?

This is to replicate something similar to `getinitialprops`, and seems to be aligned as we re-render on every search.

> And is it always pinged when those lifecycle methods fire?

Yip, including _not_ firing when we update the UI with the viewer.
